client/my-sites/site-settings/section-import.jsx

```diff
@@ -16,31 +18,26 @@ import MediumImporter from 'my-sites/importer/importer-medium';
 import { fetchState } from 'lib/importer/actions';
 import { appStates, WORDPRESS, MEDIUM } from 'state/imports/constants';
 import EmailVerificationGate from 'components/email-verification/email-verification-gate';
+import { getSelectedSite } from 'state/ui/selectors';
 
-export default React.createClass( {
-	displayName: 'SiteSettingsImport',
+class SiteSettingsImport extends Component {
+	static propTypes = {
+		site: PropTypes.object,
+	};
 
-	propTypes: {
-		site: React.PropTypes.oneOfType( [
-			React.PropTypes.bool,
-			PropTypes.shape( {
-				slug: PropTypes.string.isRequired,
-				title: PropTypes.string.isRequired
-			} )
-		] )
-	},
+	constructor( props ) {
+		super( props );
 
-	componentDidMount: function() {
+		this.state = getImporterState();
```

Comment:
I've been seeing this implemented a lot as follows: 

```js
state = getImporterState();
```

rather than a constructor function when you just need to set an initial state that doesn't depend on props. I believe they're functionally equivalent, do you think there's a benefit to using `constructor(props) { super(props); this.state = ... }` over `state = ...`?